public/scss/attackcost.scss

```diff
@@ -0,0 +1,149 @@
+input[type=number] {
+  -moz-appearance: textfield;
+  font-size: 0.95rem;
+  border-radius: 5px;
```

Comment:
Try just `3px`. Can we do something about the borders too?

![image](https://user-images.githubusercontent.com/6109680/72530250-935caa80-3834-11ea-8a02-86a5662f517f.png)